packages/store-indexer/src/sqlite/getLogs.ts

```diff
@@ -0,0 +1,39 @@
+import { Middleware } from "koa";
+import Router from "@koa/router";
+import compose from "koa-compose";
+import { input } from "@latticexyz/store-sync/indexer-client";
+import { storeTables, tablesWithRecordsToLogs } from "@latticexyz/store-sync";
+import { debug } from "../debug";
+import { createBenchmark } from "@latticexyz/common";
+import { compress } from "../compress";
+import { getTablesWithRecords } from "./getTablesWithRecords";
+import { BaseSQLiteDatabase } from "drizzle-orm/sqlite-core";
+
+export function getLogs(database: BaseSQLiteDatabase<"sync", any>): Middleware {
+  const router = new Router();
+
+  router.get("/get/logs", compress(), async (ctx) => {
```

Comment:
similar to above, would prefer to use `/api/logs` and call this function/module `api` or `apiRoutes`